### ISSUE TITLE
Add Rails Caveats Around Spring

### ISSUE THREAD

**Issue by cguess** [Nakilon/dhash-vips#10]
I think it would be a good idea to add instructions for Rails users (just to be nice), which involves removing Spring as referenced [here](https://github.com/Nakilon/dhash-vips/issues/9#issuecomment-873434274) by @jcupitt.

I can do submit a PR for this, but just wanted to make sure everyone is cool with it?

**Comment by Nakilon** [comment]
I think it's more like a ruby-vips problem. There are many other gems that depend on it.

**Comment by jcupitt** [comment]
Yes, I could well be wrong, it was just a guess. You'd need to test disabling spring, and perhaps ask a spring or senior rails dev for their opinion.

AvtiveStorage itself uses ruby-vips for image handling, so I'd expect this to be a fairly well-known issue.

**Comment by cguess** [comment]
@jcupitt Disabling spring does, in fact, fix this issue. It may be a ruby-vips issue but from my perspective it seems like a good idea to at least note it in the set up.

**Comment by jcupitt** [comment]
Ah, sorry, I hadn't realized. I agree, it's worth noting then.

I wonder how rails dodges this issue?

**Comment by cguess** [comment]
I'll ask over in the rails forums and update here when I hear more, because, yes, this seems like an interesting problem. 

In the mean time expect a PR to the docs for this.

**Comment by jcupitt** [comment]
Hello all, I think Kleis might have found the problem (or part of it) -- macOS is missing `sched_getattr` and this forces glib to start threads sooner. 

https://github.com/libvips/php-vips-ext/issues/42#issuecomment-879817757

I've pushed a change to the head of 8.11 to delay thread startup a bit longer, and this might fix this. Would anyone be able to test?

**Comment by Nakilon** [comment]
Closing since no confirmation.
